README.md

```diff
@@ -38,6 +39,26 @@ Now start your server
     ./manage.py runserver
 ```
 
+### Nix development
+For this mode of development, you must install:
+-   nix with flakes support
+-   kind
```

Comment:
Just looking at the commit history, it might be that you initialized the flake before this stuff was added to the `devenv-k8s` flake.

You should update that to the latest version using `nix flake update`